locales/en/plugin__kubevirt-plugin.json

```diff
@@ -876,6 +876,7 @@
   "Terms": "Terms",
   "The changes you have made require this VirtualMachine to be restarted.": "The changes you have made require this VirtualMachine to be restarted.",
```

Comment:
Suggest: This VirtualMachine must be restarted to apply changes.

Reply:
Thanks for the suggestion, Avital. However, this line of the code is unchanged and not related to adding help text to DataImportCron field. I'll go back to this later if needed (and fix that in a separate PR). Same for the next line and most of the other lines you've commented. The only related/changed code is the code with the green/red background in here, when looking at the code changes.

Reply:
Understood. Thanks for the explanation!